### PR TITLE
Add menhera plugin

### PR DESCRIPTION
**Pull request by Ailim**
Adds Menhera plugin, which allows using emotes listed on https://stickersurge.com/pack/menhera with command !mh or !menhera *emote*. !mh / !menhera lists all available emotes. *random* emote is also available. Available emotes are configured in the config file.

### CODE REVIEW

File: src/plugins/menhera.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	const strMap = new Map();
+	Object.keys(obj).forEach((k) => {
+		strMap.set(k, obj[k]);
+	});
+	return strMap;
+}
+
+const emotes = objToStrMap(data);
+
+function processMenheraComment(msg) {
```
Comment (AlexKolpa) [inline src/plugins/menhera.js:18]: How would you feel about dropping the `random` argument and having that be the default for `!mh/!menhara`? You could add a `!mh help` for example to get all options then.

Comment (Ailim) [reply src/plugins/menhera.js:18]: Sounds good 👍

File: src/plugins/menhera.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+function objToStrMap(obj) {
+	const strMap = new Map();
+	Object.keys(obj).forEach((k) => {
+		strMap.set(k, obj[k]);
+	});
+	return strMap;
+}
+
+const emotes = objToStrMap(data);
```
[the remark this answers is not in the document]
Comment (Ailim) [reply src/plugins/menhera.js:16]: Too much Java, not enough JS coding experience 👍

**Review by AlexKolpa — APPROVED** [review]
:shipit:

**Pull request merged**